### PR TITLE
Add a Wall

### PR DESCRIPTION
**Pull request by akshit15013**
Description : Add a wall to the Labyrinth maze 
Issue No. #813 
Preview Link : https://pr-817-fossasia-labyrinth.surge.sh/
screenshot 124

### CODE REVIEW

**Comment by akshit15013** [comment]
I HAVE A MODIFIED CODE , CAN YOU PLEASE MERGE IT?

**Comment by akshit15013** [comment]
@saarthakchats What do you mean by a preview link?

**Comment by akshit15013** [comment]
I could not get what are the conflicts

**Comment by akshit15013** [comment]
Could you help me out?

**Comment by akshit15013** [comment]
I HAVE A MODIFIED CODE , CAN YOU PLEASE MERGE IT?

File: js/tiles.js [hunk 1]
```diff
@@ -494,7 +494,6 @@ const door = {
             player.addReachableLevel(createNightLevel());
             alertWarning("Be alert!!!", "You entered to the night!");         
         },
-    }),
```
Comment (pythongiant) [inline js/tiles.js]: Why?

Comment (pythongiant) [reply js/tiles.js]: This causes your CI checks to fail

**Pull request closed without merging**